Basics/DataInventory.ipynb

```diff
@@ -0,0 +1,298 @@
+    "import lsst.afw.image as afwImage\n",
+    "import lsst.afw.table as afwTable\n",
+    "\n",
+    "import lsst.afw.display as afwDisplay"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Listing the Available Datasets\n",
+    "First, let's look at what is currently available. There are two primary shared dataset folders in the LSP, the read-only `/datasets` folder, and the group-writeable folder `/projects/shared/datasets`. Let's see what's in there"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**`/projects/shared/data`:** These datasets are designed to be small test sets, ideal for tutorials."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shared_datasets = ! ls -d /project/shared/data/* | grep -v README\n",
+    "shared_datasets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "shared_datasets=$( ls -d /project/shared/data/* | grep -v README )\n",
+    "for dataset in $shared_datasets; do\n",
+    "    du -sh $dataset\n",
+    "done"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**`/datasets`:**\n",
+    "These are typically much bigger: to measure the size, uncomment the second cell below and edit it to target the dataset you are interested in. Running `du` on all folders takes several minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datasets = ! ls -d /datasets/* | grep -v USAGE | grep -v html\n",
+    "datasets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %%bash\n",
+    "# datasets=$( ls -d /datasets/* | grep -v USAGE | grep -v html )\n",
+    "# for dataset in $datasets; do\n",
+    "#     du -h $dataset\n",
+    "# done"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualizing Sky Coverage\n",
+    "In this section, we'll plot the available patches and tracts in a given dataset on the sky, following the LSST DESC tutorial [dm_butler_skymap.ipynb](https://github.com/LSSTDESC/DC2-analysis/blob/master/tutorials/dm_butler_skymap.ipynb). In fact, we will _import_ this notebook, so that we can re-use its functions. This operation is handled by the `stackclub.wimport` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dm_butler_skymap_notebook = \"https://github.com/LSSTDESC/DC2-analysis/raw/master/tutorials/dm_butler_skymap.ipynb\"\n",
+    "\n",
+    "skymapper = stackclub.wimport(dm_butler_skymap_notebook, vb=True)"
```

Comment:
For me, this cell results in
```
WARNING: module was downloaded to {} but cound not be imported.
Returning path to module: .downloads/dm_butler_skymap.ipynb
```

and the next cell finds 0 tracts.

Reply:
I tried the notebook in v16.0 and in `w_2018_31` with the same results -- no tracts found and no plots made.

Reply:
Yup, this is a bug: I don't know why the module could not be imported. I'm thinking to comment out this cell for now, and issue a fix.